test/python/_mockutils.py

```diff
@@ -142,22 +144,18 @@ def error(self):
 
 
 def new_fake_qobj():
-    """Creates a fake qobj dictionary."""
-    return {
-        'id': 'test-id',
-        'config': {
-            'shots': 1024,
-            'max_credits': 100
-        },
-        'experiments': [{
-            'header': {'compiled_circuit_qasm': 'fake-code'},
-            'config': {
-                'seed': 123456
-            },
-            'instructions': []
-        }],
-        'header': {'backend_name': 'test-backend'}
-    }
+    """Create fake `Qobj` and backend instances."""
+    backend = FakeBackend()
+    return Qobj(
+        id='test-id',
+        config=QobjConfig(1024, 1, max_credits=100),
```

Comment:
Can you also pass the argument names in this line, in the same spirit of a previous comment?